include/rdma/fi_eq.h

```diff
@@ -75,10 +77,18 @@ struct fid_wait {
 	struct fi_ops_wait	*ops;
 };
 
+#ifndef _WIN32
 struct fi_mutex_cond {
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
 };
+#else /* _WIN32 */
+struct fi_critsec_cond {
+	CRITICAL_SECTION	*mutex;
```

Comment:
done...
i will create patch to fix windows build on monday (yea, this PR breaks windows build)

thank you